docker-postgres/Dockerfile

```diff
@@ -1,5 +1,3 @@
 FROM postgres:9.4-alpine
 
-COPY ./oisp-frontend/public-interface/deploy/postgres/base/*.sql /docker-entrypoint-initdb.d/
```

Comment:
How is the test database created without the *.sql scripts? Is this handled then later with sequelize in frontend?

Reply:
Yeah sequelize will use these scripts to create the database anyway (It was always like this, resetDB also uses these scripts)

Reply:
Should I add the TEST environment variable fix back?